src/scss/theme/_book.scss

```diff
@@ -38,14 +38,15 @@
         top: 50px;
         left: 50%;
         z-index: 5;
-        transform: translateZ(28.5px) translateX(-50%);
+        -webkit-transform: translateZ(28.5px) translateX(-50%);
     }
 
     .book-side {
         position: absolute;
         top: 299px;
         left: 50%;
-        transform: translateY(-15px) translateX(-50%) translateZ(15px) rotateX(104deg);
+        -webkit-transform: translateY(-15px) translateX(-50%) translateZ(15px)
```

Comment:
Should we specify multiple rules instead of replacing them?

Also see: https://caniuse.com/?search=transform

Reply:
Ok, rule applied for mobile device. based on https://stackoverflow.com/questions/14942081/detect-if-a-browser-in-a-mobile-device-ios-android-phone-tablet-is-used